assets/js/navbar.js

```diff
@@ -20,13 +20,22 @@ const controllerNavbar = (function (jQuery) {
       const dropdownToggle = dropdownMenu.find('.dropdown-toggle');
       const dropdownMenuContent = dropdownMenu.find('.dropdown-menu');
 
-      dropdownToggle.on('mouseenter', function () {
-        dropdownMenuContent.addClass('show');
-      })
-
-      dropdownMenu.on('mouseleave', function () {
-        dropdownMenuContent.removeClass('show');
-      })
+      // Toggle dropdown menu when clicking the dropdown toggle
+      dropdownToggle.on('click', function (e) {
+        e.preventDefault(); // Prevent default anchor behavior
+        
+        // Close all other dropdowns
```

Comment:
also this commentary is not necessary, it is clear by reading the code what is doing.